src/components/common/ProductBox/ProductBox.module.scss

```diff
@@ -77,7 +77,7 @@
 
   .line {
     margin: 0 10px;
-    border-bottom: 1px solid #2a2a2a;
+    border-bottom: 1px solid $productBox-stars;
```

Comment:
to nie są zmiany związane z tym taskiem wydaje mi sie. Zrób merge z `dev`

Reply:
Wydaje mi sie, ze to przez to, ze robilem na raz taski na dodanie zmiennych kolorow i na dodanie 'porownywarki' gdzie zmienialem style w tych komponentach i jeszcze wtedy brakowalo zmiennych. Tutaj jak zrobilem pierwszy pull w nowym branchu musialem to poprawic, zeby w ogole cokolwiek zadzialalo.

Reply:
Nie bardzo rozumiem problem. Tamte taski są już na dev i działają więc nie trzeba nic poprawiać. Wystarczy zrobić merge z dev i powinny zniknąć te zmiany.

Reply:
w momencie kiedy puszczam na tym branchu git merge dev wyskakuje mi Already up to date. Cos innego powinienem robic?